index.js

```diff
@@ -312,4 +312,14 @@ input MovePageInput {
   sectionId: ID!
   position: Int!
 }
+
+input CreateOtherAnswerInput {
+  id: ID!
+  type: AnswerType!
+}
+
+input DeleteOtherAnswerInput {
+  id: ID!
```

Comment:
Should we name this to make it obvious it's referring to the id of the "parent" answer? And if so, make it symmetrical with create